kfs-cam/src/main/java/edu/arizona/kfs/module/cab/batch/service/impl/BatchExtractServiceImpl.java

```diff
@@ -0,0 +1,97 @@
+package edu.arizona.kfs.module.cab.batch.service.impl;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.gl.businessobject.Entry;
+
+import edu.arizona.kfs.module.cab.CabConstants;
+import edu.arizona.kfs.module.purap.document.VendorCreditMemoDocument;
+
+import org.kuali.kfs.module.cab.CabPropertyConstants;
+import org.kuali.kfs.module.cab.businessobject.PurchasingAccountsPayableDocument;
+import org.kuali.kfs.module.purap.document.AccountsPayableDocumentBase;
+import org.kuali.kfs.module.purap.document.PaymentRequestDocument;
```

Comment:
Please change imports to edu.arizona.kfs.module.purap.document.PaymentRequestDocument to gain access to the payment method code added earlier in this mod.